.github/workflows/publish-pypi.yml

```
@@ -1,22 +1,19 @@
 # This workflow will upload a Python Package using Twine when a release is created
 # For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
 
-name: Upload Python Package to Test PyPI
+name: Upload Release Package to PyPI
 
 on:
   release:
     types: [created]
 
 jobs:
   deploy:
-
+    if: github.repository_owner == 'unitaryfund'
     runs-on: ubuntu-latest
-
     steps:
     - name: Check out mitiq
       uses: actions/checkout@v2
```

This has to be here I think as it instructs the release process to use the latest commit on the release branch.
```suggestion
      uses: actions/checkout@v2
      with:
        ref: 'release'
```